build.sbt

```diff
@@ -72,7 +71,11 @@ lazy val root = (project in file("."))
   .settings(uglifyCompressOptions := turnoffJSUglifyWarningsTask.value)
   .settings(
     scalacOptions := scalacOptions.value.diff(Seq("-Wunused:all")),
-    Test / scalacOptions ++= Seq(
+    scalacOptions ++= Seq("-feature",
```

Comment:
`-feature` is already set on line 52, so perhaps delete it there and keep this one?

Reply:
yeah, good spot!